design/sketch.webidl

```diff
@@ -66,20 +66,17 @@ interface WebGPUTextureView {
 };
 
 // Texture
-typedef u32 WebGPUTextureDimensionEnum;
-interface WebGPUTextureDimension {
-    const u32 e1D = 0;
-    const u32 e2D = 1;
-    const u32 e3D = 2;
-    // TODO other dimensions (cube, arrays)
-};
-
-typedef u32 WebGPUTextureFormatEnum;
-interface WebGPUTextureFormat {
-    const u32 R8_G8_B8_A8_UNORM = 0;
-    const u32 R8_G8_B8_A8_UINT = 1;
-    const u32 B8_G8_R8_A8_UNORM = 2;
-    const u32 D32_FLOAT_S8_UINT = 3;
+enum WebGPUTextureDimension {
+    "1d",
```

Comment:
this isn't going to map to the enums of native languages, since those aren't allowed to start with a number. I don't think this is a blocker, since some logic is still going to be involved in the mapping, and that logic might as well handle the leading numbers

Reply:
Agreed this is fine. Are the array types going to be in this enum as well? (I think they are - but...) if not, it may make more sense to use the actual literal values 1, 2, 3.

Reply:
Agreed that this isn't a blocker. Array-size would be a separate argument that must be 1 for non-array or N > 1 for array. I thought about making it a number but it is a significant departure from other APIs and seems like we could have extensions needing to add dimensions in the future.

Reply:
Yeah I wasn't clear, I meant that 2D textures are always arrays potentially of size one.